libs/blocks/caas/utils.js

```diff
@@ -16,17 +16,49 @@ const fetchWithTimeout = async (resource, options = {}) => {
   return response;
 };
 
-export const loadStrings = async (url) => {
-  // TODO: Loc based loading
+export const locales = ['ae_ar', 'ae_en', 'africa', 'ar', 'au', 'be_en', 'be_fr', 'be_nl', 'bg', 'ca', 'ca_fr', 'ch_de',
+  'ch_fr', 'ch_it', 'cis_en', 'cl', 'cn', 'cy_en', 'de', 'es', 'fr', 'gr_en', 'hk_en', 'id_en', 'ie', 'il_en', 'in',
+  'jp', 'kr', 'la', 'langstore', 'lu_de', 'lu_en', 'lu_fr', 'lv', 'mena_ar', 'mena_en', 'mt', 'mx', 'my_en', 'nl',
+  'nz', 'pe', 'ph_en', 'pt', 'sa_ar', 'sa_en', 'se', 'sea', 'sg', 'th_en', 'uk', 'vn_en'];
+
+export function getPageLocale(currentPath){
+  let possibleLocale = currentPath.split('/')[1];
+  for(let locale of locales){
+    if(locale === possibleLocale){
+      return locale;
+    }
+  }
+  // defaults to en_US
+  return '';
+}
+
```

Comment:
The `config` object provided by `getConfig` (in utils/utils.js) provides:
`config.locale` which is the current locale
`config.locales` which is a list of all supported locales
You can use that to get the localized path to the strings doc.